tox.ini

```diff
@@ -1,5 +1,16 @@
 [tox]
-envlist = py26, py27, py32, py33, py34
+envlist = py26, py27, py32, py33, py34, flake, flake_docs
 
 [testenv]
 commands = python tests.py
+
+[testenv:flake]
```

Comment:
I totally just realized this isn't doing what I thought it was, let me look into it.